docs/pages/setup/reference/cli.mdx

```diff
@@ -311,6 +311,40 @@ $ tsh play --format=pty 1fe153d1-ce8b-4ef4-9908-6539457ba4ad.tar
 $ tsh play --format=json ~/play/0c0b81ed-91a9-4a2a-8d7c-7495891a6ca0.tar | jq '.event
 ```
 
+### tsh proxy app
+
+Start a local TLS proxy for Application Access connections when using Teleport in single-port mode.
+
+```code
+$ tsh proxy app [<flags>] <app>
+```
+
+#### Arguments
+`<app>`
+
+- `app` The name of the application to start the local proxy for. To see a list of available applications, run `tsh apps ls`.
+
+#### Flags
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
+| - | - | - | - |
+| `--port` | none | port | Specify the source port for the local proxy |
+
+#### [Global Flags](#tsh-global-flags)
+
+These flags are available for all commands `--login, --proxy, --user, --ttl, --identity, --cert-format, --insecure, --auth, --skip-version-check, --debug, --jumphost, --format`.
+Run `tsh help <subcommand>` or see the [Global Flags Section](#tsh-global-flags)
+
+#### Examples 
+```code
+$ tsh --proxy proxy.example.com:443 proxy app <app>
```

Comment:
nit: the proxy address by default will be taken from take from `tsh` user profile. 
I don't see purpose of `--proxy proxy.example.com:443` flag in `tsh proxy app` and at first glance it make the command example harder to grasp.

Reply:
Agreed. Fixed in https://github.com/gravitational/teleport/pull/10924/commits/f3b68d2a1610cbfb32a9536471d1fcddccd8b899